Formula/glibc.rb

```diff
@@ -195,4 +203,13 @@ def post_install
     safe_system "#{lib}/libc.so.6", "--version"
     safe_system "#{bin}/locale", "--version"
   end
+
+  def ld_so_conf
+    <<~EOS
+      include #{etc}/ld.so.conf.d/*.conf
+      include #{opt_prefix}/etc/ld.so.conf.d/*.conf
```

Comment:
```suggestion
      include #{prefix}/etc/ld.so.conf.d/*.conf
```
You write this file inside `prefix`, so it can refer directly to `prefix`